websocket.go

```diff
@@ -19,23 +19,42 @@ var WsProtocol = ma.Protocol{
 	Name:  "ws",
 	VCode: ma.CodeToVarint(477),
 }
+var WssProtocol = ma.Protocol{
+	Code:  478,
+	Name:  "wss",
+	VCode: ma.CodeToVarint(478),
+}
 
-var WsFmt = mafmt.And(mafmt.TCP, mafmt.Base(WsProtocol.Code))
+var WsFmt = mafmt.Or(
+	mafmt.And(mafmt.TCP, mafmt.Base(WsProtocol.Code)),
+	mafmt.And(mafmt.TCP, mafmt.Base(WssProtocol.Code)),
```

Comment:
It might be more readable as `TCP and (WS or WSS)` instead of `(TCP and WS) or (TCP and WSS)`